modules/appeals_api/spec/docs/supplemental_claims/v0_spec.rb

```diff
@@ -222,33 +192,6 @@ def swagger_doc
         }
       }
 
```

Comment:
I pushed one small update - I had forgotten to remove these parameter definitions for the validate endpoint, even though they aren't used anymore